Libraries/LibWeb/CSS/Parser/Parser.h

```diff
@@ -293,7 +293,7 @@ class Parser {
     Optional<PropertyAndValue> parse_css_value_for_properties(ReadonlySpan<PropertyID>, TokenStream<ComponentValue>&);
     RefPtr<CSSStyleValue> parse_builtin_value(TokenStream<ComponentValue>&);
     RefPtr<CSSStyleValue> parse_calculated_value(ComponentValue const&);
-    RefPtr<CustomIdentStyleValue> parse_custom_ident_value(TokenStream<ComponentValue>&, std::initializer_list<StringView> blacklist);
+    RefPtr<CustomIdentStyleValue> parse_custom_ident_value(TokenStream<ComponentValue>&, Span<StringView> blacklist);
```

Comment:
`ReadonlySpan`

Reply:
One of these days I will remember that `ReadonlySpan` exists. :sweat_smile: Thanks!